lightkurve/lightcurvefile.py

```diff
@@ -201,6 +195,10 @@ def channel(self):
     def keplerid(self):
         return self.header(ext=0)['KEPLERID']
 
+    @property
+    def cadence(self):
```

Comment:
Let's call this `obsmode` instead of `cadence` for consistency with header keyword conventions.